activerecord/lib/active_record/attribute_methods.rb

```diff
@@ -352,23 +352,22 @@ def []=(attr_name, value)
     #
     # For example:
     #
-    #   class PostsController < ActionController::Base
+    #   class PostsController < ApplicationController
     #     after_action :print_accessed_fields, only: :index
     #
     #     def index
     #       @posts = Post.all
     #     end
     #
     #     private
-    #
-    #     def print_accessed_fields
-    #       p @posts.first.accessed_fields
-    #     end
+    #       def print_accessed_fields
```

Comment:
"Indent and no blank line after private/protected." style is for Ruby on Rails framework. Not all Rails applications need to follow this style, therefore I prefer to restore the original document here.

Reply:
Refer to https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#follow-the-coding-conventions

> Indent and no blank line after private/protected.

Reply:
Hey!
Maybe I'm missing something but to me looks like that I've followed the rails documentation.
Old example was not indented and had a blank line after private, this changes it to match the convention (ident without blank line).